applications/luci-app-p910nd/root/usr/share/rpcd/acl.d/luci-app-p910nd.json

```diff
@@ -1,11 +1,34 @@
 {
 	"luci-app-p910nd": {
-		"description": "Grant UCI access for luci-app-p910nd",
+		"description": "Grant UCI access for luci-app-p910nd to hotplug.d in case hotplug scripts are necessary.",
 		"read": {
-			"uci": [ "p910nd" ]
+			"cgi-io": [ "exec" ],
+			"uci": [ "p910nd" ],
+			"file": {
+				"/bin/mkdir -p": [ "exec" ],
+				"/bin/mkdir": [ "*" ],
+				"/dev/usb/": ["read"],
+				"/etc/hotplug.d/usb/": [ "read" ],
+				"/etc/hotplug.d/usb/*": [ "read" ],
+				"/etc/hotplug.d/usbmisc/": [ "read" ],
+				"/etc/hotplug.d/usbmisc/*": [ "read" ],
+			},
+			"ubus": {
+				"luci": [ "p910nd" ],
+				"file": [ "stat" ]
+			}
 		},
 		"write": {
-			"uci": [ "p910nd" ]
+			"cgi-io": [ "exec" ],
+			"uci": [ "p910nd" ],
+			"file": {
+				"/bin/mkdir -p": [ "exec" ],				
+				"/bin/busybox mkdir -p": [ "exec" ],
```

Comment:
This ACL rule likely never matches, it should be `/bin/busybox mkdir -p *` - apart from that authenticated users should not be able to create directories with root permissions anywhere in the system, it should be dropped.